### PR TITLE
Fixes #2484 - Error message when unable to save file

### LINKED ISSUE #2484: Selecting inaccessible location while accepting incoming file breaks qTox's UI.
It shows weird inline message which cannot be closed.

pic

Whole qTox UI is blocked, even systray icon.

I still see new incoming messages and "typing..." notifications. The message get shifted upwards when new messages arrives.

### PR DESCRIPTION
**Pull request by a68366**
Show proper error message. qTox is not locked.

### CODE REVIEW

**Comment by ghost** [comment]
Works fine.

Debian testing
Cinnamon 2.6.13
Qt 5.5.1

File: src/chatlog/content/filetransferwidget.cpp [hunk 1]
```diff
@@ -138,8 +139,7 @@ void FileTransferWidget::acceptTransfer(const QString &filepath)
     //test if writable
     if (!Nexus::tryRemoveFile(filepath))
     {
-        QMessageBox::warning(this,
-                             tr("Location not writable", "Title of permissions popup"),
+        GUI::showWarning(    tr("Location not writable", "Title of permissions popup"),
```
Comment (zetok) [inline src/chatlog/content/filetransferwidget.cpp]: Spaces between `showWarning(` and `tr(` should be removed

**Comment by zetok** [comment]
@a68366 Since it works as it should, as soon as spaces will be removed, please comment so that PR could be merged `:)`

Thanks for the fix `:)`

**Comment by a68366** [comment]
@zetok Done

**Pull request merged**